app/build.gradle

```diff
@@ -27,6 +27,6 @@ dependencies {
 //    implementation "com.algolia.instantsearch-android:voice:1.+"
     implementation project(":instantsearch.voice")
 
-    implementation 'com.android.support:appcompat-v7:28.0.0-alpha1'
+    implementation 'com.android.support:appcompat-v7:28.0.0-rc01'
```

Comment:
Here you use 28.0.0, below I see 27.1.1

Reply:
Discussed offline, demo uses latest while lib uses latest _stable_